lib/formats/map.js

```diff
@@ -3,6 +3,17 @@ const { fileHeader } = StyleDictionary.formatHelpers;
 
 const makeEntries = dictionary => dictionary.allTokens.map(x => [`--${x.name}`, x.value]);
 
+
+const makeMetaEntries = dictionary => dictionary.allTokens.map(x =>
+  [`--${x.name}`,
```

Comment:
I guess that would depend on if we chose to extend later instead of having an additive change a change to the properties themselves would just reflect here?  I didn't think at current state any of the other properties made sense outside the tokens project itself, but maybe in the context of future additions/changes just dumping the whole object just makes more sense then only value, description and extensions being filtered.

Reply:
design tokens have (or at least, are meant to have) a set schema, so we might as well just offer the whole token

we already ship the nested json structure in tokens.json. this module is a convenience to get tokens by name.